src/components/App.js

```diff
@@ -1,28 +1,81 @@
 import React, { useState, useEffect } from 'react';
-
+import ReactDOM from "react-dom";
+import { Routes, Route, NavLink, BrowserRouter as Router } from "react-router-dom";
```

Comment:
we don't need to import `BrowserRouter as Router`

Reply:
complete